plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-order-summary-block/frontend.tsx

```diff
@@ -50,77 +54,89 @@ const FrontendBlock = ( {
 	// rendered on small and mobile screens.
 	return (
 		<>
-			<div className={ className }>
-				<div
-					className={ clsx(
-						'wc-block-components-checkout-order-summary__title',
-						{
-							'is-open': isOpen,
-						}
-					) }
-					{ ...orderSummaryProps }
-				>
-					<p
-						className="wc-block-components-checkout-order-summary__title-text"
-						role="heading"
+			<LoadingMask
+				isLoading={
+					isApplyingExtensionCartUpdate ||
+					isLoadingRates ||
+					isSelectingRate
+				}
+				screenReaderLabel={ __(
+					'Loading shipping rates…',
+					'woocommerce'
+				) }
```

Comment:
Nitpic: I wonder if we can find a better phrase than _"Loading shipping rates…"_, as the spinner shows both when loading and selecting shipping rates and local pickup methods.

Reply:
Well, this shouldn't be shipping rates at all, this is on the order summary, I updated it. Thanks Niels.